### ISSUE TITLE
error: package directory 'dephell/repositories/conda' does not exist 

### ISSUE THREAD

**Issue by CircleOnCircles** [dephell/dephell#154]
Continue from #152 , There is still another error raise.
I will create a PR to fix this old setup.py soon.

```
[EnvCommandError]                                                                                  
Command ['python', '-m', 'pip', 'install', '-e', '/home/cocs/development/OpenSource/dephell'] errored with the                
 following output:                                                                                              
Obtaining file:///home/cocs/development/OpenSource/dephell                                                      
    ERROR: Complete output from command python setup.py egg_info:                                               
    ERROR: running egg_info                                                                                     
    writing dephell.egg-info/PKG-INFO                                                                           
    writing dependency_links to dephell.egg-info/dependency_links.txt                                           
    writing entry points to dephell.egg-info/entry_points.txt                                                   
    writing requirements to dephell.egg-info/requires.txt                                                       
    writing top-level names to dephell.egg-info/top_level.txt                                                   
    error: package directory 'dephell/repositories/conda' does not exist                                          
    ----------------------------------------                                                                    
ERROR: Command "python setup.py egg_info" failed with error code 1 in /home/cocs/development/OpenSource/dephel  
l/  
```

**Comment by CircleOnCircles** [comment]
I think we should add [pre-commit](https://pre-commit.com/) hook to prevent a future error like this.

**Comment by orsinium** [comment]
Command project build updates setup.py on every release.

Maybe, pre-commit hook is suitable here, right. BTW, we have recept for this:
https://dephell.readthedocs.io/en/latest/use-git-hook.html

**Comment by CircleOnCircles** [comment]
I will put some notes here about another error found in sequences just for a reference but it isn't responsible for this library. I filed it here sdispater/poetry#1164 .

The error msg is

```bash
▶ poetry show -vvv     
Using virtualenv: /home/cocs/.cache/pypoetry/virtualenvs/dephell-py3.7
                                    
[ParseVersionError]  
Unable to parse "discover)".  
```

This error happens after the successful installation. So, just ignore it until the source library is fixed.

**Comment by orsinium** [comment]
No-no, it's my fault. I've uploaded broken `dephell-discover`. Please, update `dephell_discover` to `0.2.6`. See #117 for background

**Comment by CircleOnCircles** [comment]
> No-no, it's my fault. I've uploaded broken `dephell-discover`. Please, update `dephell_discover` to `0.2.6`. See #117 for background

I see. I closed the issue sdispater/poetry#1164 . and will update the `pyproject.toml` `dephell-discover  = ">=0.2.6"`.

**Comment by orsinium** [comment]
Closed in #159
